### PR TITLE
Add blueprint tests for ember-data blueprints

### LINKED ISSUE #9: Add blueprint tests for ember-data blueprints
Currently we are missing blueprint tests for ember-data. We need to add blueprint tests for:
- [ ] adapter
- [ ] adapter-test
- [ ] model
- [ ] model-test
- [ ] serializer
- [ ] serializer-test
- [ ] transform
- [ ] transform-test

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves #9

### CODE REVIEW

**Comment by stefanpenner** [comment]
@homu r+

**Comment by trabus** [comment]
I'm confused on how the `throws` tests are passing now without [this fix](https://github.com/ember-cli/ember-cli-blueprint-test-helpers/pull/35). They were not passing in ember-data previously, was there some work done to fix this?

File: node-tests/blueprints/adapter-test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          contains: [
+            'moduleFor(\'adapter:foo\''
+          ]
+        }
+      ]
+    });
+  });
+
+  // skipped because `throws` is currently broken
+  it.skip('adapter throws when --base-class is same as name', function() {
```
Comment (Turbo87) [inline node-tests/blueprints/adapter-test.js]: @trabus see above :wink:

**Comment by trabus** [comment]
@Turbo87 ah, missed that skip. Thanks!

**Pull request merged**